### ISSUE TITLE
Error creating issue 

### ISSUE THREAD

**Issue by jagguli** [ahungry/org-jira#220]
```[error] request--curl-sync: semaphore never called
Fetching issues...
helm-M-x-execute-command: Invalid slot type: org-jira-sdk-issue, created, string, nil
```

Emacs version 26.3

**Comment by ahungry** [comment]
Can you add some more info?  What command was executing as this error occurred?

**Comment by Viglioni** [comment]
I had the same problem when I assigned it to "Unassigned"

With these configs:
```
(setq request-log-level 'debug)
(setq request-message-level 'debug)
```

I've got when i call `org-jira-create-issue`:

```
{"errorMessages":[],"errors":{"assignee":"Account ID  does not exist."}}
[error] request--callback: peculiar error: 400
[debug] request--callback: executing error
```

but when I've assigned to a valid user I had no problem.

I'm not sure if it is a bug or if the API does not allow if I discover anything about this I comment here

**Comment by fridim** [comment]
I can't make that work, even when i set an Assignee, i get the same error:

```
{"errorMessages":[],"errors":{"assignee":"expected Object containing a 'name' property"}}
[error] request--callback: peculiar error: 400
[debug] request--callback: executing error
[error] request-default-error-callback: https://[...EDITED...]/rest/api/2/issue error
jiralib--rest-call-it: Wrong type argument: stringp, nil
```

**Comment by noorul** [comment]
I am also getting the same error

**Comment by ahungry** [comment]
Should be fixed in 0e9bb6d0bc9241bdee72a1c6c78472f4cf30965b lemme know if not.
